ui/docker/Dockerfile

```diff
@@ -2,9 +2,12 @@
 FROM node:14.15.4 as node
 WORKDIR /app
 COPY package.json /app/
-RUN npm install
+# I'm not sure if yarn is available from this node it imported
```

Comment:
yes it's available, let's remove these comments and then we can land it

Reply:
> yes it's available, let's remove these comments and then we can land it

:+1:

Reply:
You're right, it does indeed come pre-installed.
@daitan-rovaris I've just removed the --force;
After running both alternatives and checking yarn versions available, it seemed to be overkill.